vms/platformvm/vm_test.go

```diff
@@ -2189,3 +2177,14 @@ func TestPruneMempool(t *testing.T) {
 	_, ok = vm.Builder.Get(baseTxID)
 	require.True(ok)
 }
+
+func createSubscriberAndChangeNotifier() (*smblock.ChangeNotifier, common.Subscription) {
+	var cn smblock.ChangeNotifier
+
+	subscription := func(ctx context.Context) (common.Message, error) {
+		<-ctx.Done()
+		return common.Message(0), ctx.Err()
+	}
+
+	return &cn, subscription
+}
```

Comment:
Sorry I forgot to comment on this function in the prior review as well. This guy should be cleaned up with the same `noopSubscription`